src/rules/prefer-flowmax.coffee

```diff
@@ -0,0 +1,68 @@
+{isFlow, isMagic, isBranchPure, getFlowToFlowMaxFixer, isNonmagicHelper, isFunction} = require '../util'
+
+module.exports =
+  meta:
+    docs:
+      description: 'Flag flow() calls that could be converted to flowMax()'
+      category: 'Possible Errors'
+      recommended: yes
+    schema: [
+      enum: ['always', 'whenUsingUnknownHelpers']
+    ,
+      type: 'object'
+      properties:
+        shouldFix:
```

Comment:
Also support `shouldFix` like `needs-flowmax` (for `known-imports` autofixing `flow()` -> `flowMax()`)